backend/services/iot-manager/store/mongo/migrations.go

```diff
@@ -26,7 +26,7 @@ import (
 
 const (
 	// DbVersion is the current schema version
-	DbVersion = "1.1.1"
+	DbVersion = "1.1.2"
```

Comment:
One small request, can we bump the minor version since this migration is required for the service to remain backwards compatible?

Reply:
sure. 😘